ClientCore/Enums/ClientTypeHelper.cs

```diff
@@ -1,15 +1,23 @@
 ﻿using System;
+using System.Linq;
+using ClientCore.Extensions;
 
 namespace ClientCore.Enums
 {
     public static class ClientTypeHelper
     {
+        private const ClientType ct = 0;
         public static ClientType FromString(string value) => value switch
         {
             "TS" => ClientType.TS,
             "YR" => ClientType.YR,
             "Ares" => ClientType.Ares,
-            _ => throw new Exception("It seems the client configuration was not migrated to accommodate for the v2.12 changes. Please specify 'ClientGameType' in `[Settings]` section of the 'ClientDefinitions.ini' file, e.g., 'ClientGameType=Ares'."),
+            _ => throw new Exception(string.Format(("It seems the client configuration was not migrated to accommodate for the v2.12 changes. " +
+                                                    "Please specify 'ClientGameType' in '[Settings]' section of the 'ClientDefinitions.ini' file " +
+                                                    "(allowed options: {0}).\n\n" +
+                                                    "Please refer to documentation of the client {1} for more details. This link can also be found in the log file.").L10N("Client:Main:ClientGameTypeNotFoundException"),
+                                                    ct.GetNames(),
```

Comment:
String.Join

Reply:
> String.Join

Why are you recommend to use `string.Join` instead of `+`, if client's code have plenty of such examples?

Also, if i use `string.Join` VS generates new warning:

![image](https://github.com/user-attachments/assets/4d65d608-9fd7-40c2-9fae-c44733f8519d)

Reply:
From your screenshot, I need to remind you that L10N should be called from a literal string, not a result from string.Join